packages/legend-query/src/stores/QueryBuilderExplorerState.ts

```diff
@@ -99,6 +103,7 @@ export class QueryBuilderExplorerTreeRootNodeData extends QueryBuilderExplorerTr
 export class QueryBuilderExplorerTreePropertyNodeData extends QueryBuilderExplorerTreeNodeData {
   property: AbstractProperty;
   parentId: string;
+  fromSubClass: boolean;
```

Comment:
model-wise, it is probably better to have `subClass?: Class | undefined` instead